integration-cli/fixtures_windows_daemon.go

```diff
@@ -0,0 +1,56 @@
+package main
```

Comment:
I'm pretty sure none of this is needed on Windows (to Windows) CI any more. WindowsRS1 CI runs through PowerShell now for the most part, and doesn't invoke what was `.ensure-frozen-images-windows`.